ui/pages/confirmations/utils/network.ts

```diff
@@ -7,3 +9,14 @@ export function getImageForChainId(chainId: string): string | undefined {
     ...MULTICHAIN_TOKEN_IMAGE_MAP,
   }[chainId];
 }
+
+export const isTronChainId = (chainId: Hex | number | CaipChainId | string) => {
+  if (isCaipChainId(chainId)) {
+    return [
+      TrxScope.Mainnet.toString(),
+      TrxScope.Nile.toString(),
+      TrxScope.Shasta.toString(),
+    ].includes(chainId);
+  }
+  return false;
```

Comment:
Nit: Cosmetic: This could have been written using 1 single-expression

```suggestion
  return isCaipChainId(chainId) &&
    [
      TrxScope.Mainnet.toString(),
      TrxScope.Nile.toString(),
      TrxScope.Shasta.toString(),
    ].includes(chainId);
```